mappings/net/minecraft/server/world/ServerChunkManagerEntry.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS tz net/minecraft/server/world/ServerChunkManagerEntry
+	CLASS tz$a Unloaded
+		FIELD a INSTANCE Ltz$a;
+	CLASS tz$b
+	FIELD a UNLOADED_CHUNK Lcom/mojang/datafixers/util/Either;
+	FIELD b UNLOADED_CHUNK_FUTURE Ljava/util/concurrent/CompletableFuture;
+	FIELD c UNLOADED_WORLD_CHUNK Lcom/mojang/datafixers/util/Either;
+	FIELD d UNLOADED_WORLD_CHUNK_FUTURE Ljava/util/concurrent/CompletableFuture;
+	FIELD e chunkStatuses Ljava/util/List;
+	FIELD j chunk Ljava/util/concurrent/CompletableFuture;
+	FIELD m players Luh;
+	FIELD n pos Lbag;
+	FIELD o blockUpdatePositions [S
+	FIELD p blockUpdateCount I
+	FIELD u lightingProvider Lcdq;
+	METHOD a markForUpdate (III)V
+	METHOD a sendBlockEntityUpdatePacket (Lbas;Let;I)V
+	METHOD a flushUpdates (Lbql;I)V
+	METHOD a updateChunk (Ljava/util/concurrent/CompletableFuture;)V
+	METHOD a sendPacket (Ljp;ZI)V
+	METHOD c getChunk ()Lbql;
+	METHOD f getChunk ()Ljava/util/concurrent/CompletableFuture;
```

Comment:
Double method name. I don't believe this can compile?

Reply:
@asiekierka Referring to the current name? (getChunkAsync)

Reply:
Referring to the one below bieng a bridge. This looks like METHOD c and METHOD f having conflicting signatures...

Reply:
Well, not anymore

Reply:
Okay, great